og.install

```diff
@@ -93,3 +94,48 @@ function og_update_8002() {
   $manager->updateEntityType($entity_type);
   $manager->updateFieldStorageDefinition($manager->getFieldStorageDefinition('uuid', 'og_membership'));
 }
+
+/**
+ * Add auto_add_group_owner_membership configuration to og.settings config.
+ */
+function og_update_8003(&$sandbox) {
+  $config = \Drupal::configFactory()->getEditable('og.settings');
+  // This setting defaults to TRUE for backwards compatibility.
+  $config->set('auto_add_group_owner_membership', TRUE);
+  $config->save(TRUE);
+}
+
+/**
+ * Add "No members in the group yet" empty result text to members overview view.
+ */
+function og_update_8004(&$sandbox) {
+  if (!\Drupal::service('module_handler')->moduleExists('views')) {
+    // Views is not enabled in this site. Nothing to do.
+    \Drupal::logger('og')->info('The views module is not enabled. Update 8004 was skipped.');
+    return;
+  }
+  $view = Views::getView('og_members_overview');
+  if (empty($view)) {
+    \Drupal::logger('og')->warning('The view og_members_overview does not exist. Update 8004 was skipped.');
+    return;
+  }
+  // Update the view empty text.
+  $item = [
+    'id' => 'area',
+    'table' => 'views',
+    'field' => 'area',
+    'relationship' => 'none',
+    'group_type' => 'group',
+    'admin_label' => 'No results text',
+    'empty' => TRUE,
+    'tokenize' => FALSE,
+    'content' => [
+      'value' => 'No members in the group yet.',
+      'format' => 'basic_html',
```

Comment:
Is it ok to assume basic_html is an existing format here? (Also same in the optional installation view config) 

If not, does anyone have any pointers on how this is usually handled? Perhaps I should use a different plugin/field?

Reply:
I don't think we need an upgrade path for Views config.

Reply:
> I don't think we need an upgrade path for Views config.

@amitaibu so we don't want to update existing installations and won't need this update hook? I'm not sure I understand exactly what you mean.

Reply:
Correct, no upgrade hook for Views

Reply:
Thanks! I'll remove

Reply:
FYI I'm also setting the empty text to unfiltered text in the installationg config yml instead of trying to force text formats